fastlane/metadata/android/ar/full_description.txt

```diff
@@ -1,45 +1,40 @@
-يضع تطبيق ووردبريس لنظام التشغيل Android قوة النشر على الويب في متناولك. إنَّه مُنشئ موقع على الويب وأكثر من ذلك بكثير!
```

Comment:
Submission to PlayStore Console confirmed that this and all the other `fastlane/metadata/android/*/full_description.txt` for other locales have been taken into account:
![PlayStore-20 6](https://user-images.githubusercontent.com/216089/188202624-eecf0273-c2e1-4a78-acdf-b15303f8f009.png)

Reply:
Ah! That made me realize that "English — en-US" was not part of that list 😱  — same but as for the release notes above I guess ➡️  [Follow up below](https://github.com/wordpress-mobile/WordPress-Android/pull/17118#issuecomment-1235735651)